content/en/database_monitoring/guide/tag_database_statements.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Tagging SQL Statements
+kind: guide
+---
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+This guide assumes that you have configured [Database Monitoring][1].
+
+[Datadog Database Monitoring (DBM)][1] allows you to view explain plans and query samples running on your database hosts. This guide shows you how to add tags as SQL comments to your database queries, which can then be surfaced and leveraged in DBM.
+
+## Before you begin
+
+Supported databases
+: postgres, mysql, sqlserver
+
+Supported Agent versions
+: 7.36.1+
+
+Supported tagging formats
+: [sqlcommenter][2], [marginalia][3]
+
+
+## Manual tag injection
+Using any database API supporting execution of SQL statements, add a comment in your statement with tags formatted as per the [sqlcommenter][2] or [marginalia][3] formats.
+
+```sql
+/*key='val'*/ SELECT * from FOO
+```
+
+Full example:
+```go
+import (
+	"database/sql"		
+)
+
+func main() {	
+	db, err := sql.Open("postgres", "postgres://pqgotest:password@localhost/pqgotest?sslmode=disable")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Tag SQL statement with key:val
+	rows, err := db.Query("/*key='val'*/ SELECT * from FOO")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer rows.Close()
+}
+```
+
+## Explore the tags in DBM
+
+On the [**DBM > Samples**][4] page, filter the **Explain Plans** and **Query Samples** views by custom tag.
+
+{{< img src="database_monitoring/dbm_filter_explain_plans_by_custom_tag.png" alt="Filter explain plans by custom tag.">}}
+
+You can also view a timeseries of explain plan costs filtered by tag.
```

Comment:
What is `num` in this image?

Reply:
It's the custom tag that we inject but it doesn't really any useful meaning. I would have liked something closer to a real use-case but we decided that changes to the demo app were out of scope for this.

Reply:
Updated to use `key` instead.